src/executables/uber-time.js

```diff
@@ -9,12 +9,22 @@ import CommandExecutionService from '../services/CommandExecutionService';
 const service = new CommandExecutionService();
 
 program
-  .arguments('<address>')
-  .action((address) => {
-    try {
-      service.executeTimeEstimates(address).then(table => console.log(table));
-    } catch (Error) {
-      console.error('Could not get time estimates');
-    }
-  })
+  .usage('<address>')
+  .description('Get Time-To-Pickup Estimates')
   .parse(process.argv);
+
+const address = program.args.toString();
```

Comment:
Ah, I didn't see why the trim case was necessary. I think there's a better way of capturing this exception than trim though. What kind of JSON does it respond with? I'll investigate.